src/content/hr/guides/json-parameters/readme.md

```diff
@@ -0,0 +1,38 @@
+Below is a list of the possible JSON parameters that can be included within the JSON files generated during the [FBX to GR2][hr-tool#fbx-to-gr2] process. Note that in order for any changes to JSON files to take effect, you'll need to save your changes and re-run `fbx-to-gr2`.
+
+NOTE: When manually editing a JSON, all properties should feature the prefix `bungie_`
```

Comment:
change this into an alert and it's all good

Reply:
Added alert formatting